tests/test.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+- hosts: all
+  become: True
+  roles:
+    - role: ansible-midonet-repo
+      midonet_version: 'midonet-5.2'
+      mem_username: 'calsoft'
+      mem_password: '3adfF3cd'
```

Comment:
CI jobs will need to pass the valid credentials somehow, we can not expose the real credentials on a public github repository. Change it.